birth-registration/fhir-document.jsonc

```diff
@@ -595,6 +595,32 @@
       }
     },
 
+    // Obs - Place of birth
```

Comment:
The birth encounter already has a location which we could use for place of birth and place or death. I think we should use those rather. It's more fitting.